bootstrap.sh

```diff
@@ -17,4 +17,4 @@ kubectl apply -f .infrastructure/app/deployment.yml
 
 # Install Ingress Controller
 kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
-# kubectl apply -f .infrastructure/ingress/ingress.yml
+kubectl apply -f .infrastructure/ingress/ingress.yml
```

Comment:
Verify that the path '.infrastructure/ingress/ingress.yml' is correct. It should match the directory structure where the Ingress configuration file is located. If the path is incorrect, it will result in an error when applying the Ingress configuration.